pandas/core/arrays/period.py

```diff
@@ -148,7 +148,7 @@ class PeriodArray(PeriodMixin, dtl.DatelikeOps):
     # --------------------------------------------------------------------
     # Constructors
 
-    def __init__(self, values, freq=None, dtype=None, copy=False):
+    def __init__(self, values, dtype=None, freq=None, copy=False):
```

Comment:
we should consider making optional values keyword only generally (let's create an issue / discussion point)